projects/_posts/2024-06-22-ASD-Diganosis.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: project
+title: "ASD-Diagnosis: Developing Novel Deep Learning Models for ASD Diagnosis using Complex MRI Data"
```

Comment:
How about "Machine-learning models for diagnosis and characterization of Neurodisorders using complex MRI data" 

OR 
Machine-learning models for diagnosis and characterization of Autism Spectrum Disorder using complex MRI data